src/NServiceBus.Gateway/Gateway.cs

```diff
@@ -85,9 +82,9 @@ protected override void Setup(FeatureConfigurationContext context)
                 channelManager,
                 channelReceiverFactory,
                 GetEndpointRouter(context),
-                b.Build<IDispatchMessages>(),
+                b.GetRequiredService<IDispatchMessages>(),
                 storageConfiguration.CreateStorage(b),
-                b.BuildAll<IDataBus>()?.FirstOrDefault(),
+                b.GetServices<IDataBus>()?.FirstOrDefault(),
```

Comment:
I'm not sure `GetServices` can ever return `null` but we'd have to check the specs I guess.